test/k8s-integration/cluster.go

```diff
@@ -379,3 +394,10 @@ func getKubeClient() (kubernetes.Interface, error) {
 	}
 	return kubeClient, nil
 }
+
+func isGKEDeploymentInstalledByDefault(clusterVersion string) bool {
+	cv := apimachineryversion.MustParseSemantic(clusterVersion)
```

Comment:
Since this is GKE version, should we use [this](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/423ae53cb07dea2ad3a9659bc92df8a303a5503b/vendor/k8s.io/apimachinery/pkg/util/version/version.go#L116)?

Reply:
That's the one used here I believe

Reply:
Check this https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/k8s-integration/main.go#L530 generateGKETestSkip()

Reply:
ah gotcha, updated